src/components/icons/assets/alert.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Props } from '../icon';
+
+export function AlertIcon(extraProps: Props) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" {...extraProps}>
+      <path
+        fill-rule="evenodd"
```

Comment:
```suggestion
        fillRule="evenodd"
```

This error appears when using the dash-delimited version:
```Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?```